upcloud_api/cloud_manager/storage_mixin.py

```diff
@@ -53,13 +54,20 @@ def create_storage(
         """
         if backup_rule is None:
             backup_rule = {}
+
+        if encrypted:
+            encrypted_str = 'yes'
+        else:
+            encrypted_str = 'no'
```

Comment:
code style nitpick: i like inline if 😄 

```suggestion
        encrypted_str = 'yes' if encrypted else 'no'
```